go/admin_getbucketsessionid.go

```diff
@@ -0,0 +1,25 @@
+package bucketclient
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+)
+
+// AdminGetBucketSessionID returns the raft session ID of the given bucket.
+// Returns -1 and an error if the bucket doesn't exist, or if a request error occurs.
+func (client *BucketClient) AdminGetBucketSessionID(ctx context.Context, bucketName string) (int, error) {
+	resource := fmt.Sprintf("/_/buckets/%s/id", bucketName)
+	responseBody, err := client.Request(ctx, "AdminGetBucketSessionID", "GET", resource)
+	if err != nil {
+		return -1, err
+	}
+	sessionId, err := strconv.ParseInt(string(responseBody), 10, 0)
+	if err != nil {
+		return -1, &BucketClientError{
+			"AdminGetBucketSessionID", "GET", client.Endpoint, resource, 0, "",
+			fmt.Errorf("bucketd did not return a valid session ID in response body"),
+		}
+	}
+	return int(sessionId), nil
```

Comment:
Is this type cast needed ? :thinking:

Reply:
Yes, because `strconv.ParseInt` returns an `int64`. I could as well have `AdminGetBucketSessionID` return an `int64` too, although I like the caller not having to deal with the integer size when it doesn't really matter (raft session IDs will always be way smaller than the int limit anyway).